src/plugins/analysis/ip_and_uri_finder/view/ip_and_uri_finder.html

```diff
@@ -3,7 +3,7 @@
 {% block analysis_result_details %}
 
 	{% for key in firmware.processed_analysis[selected_analysis] %}
-		{% if key not in ['summary', 'plugin_version', 'analysis_date', 'skipped'] %}
+		{% if key not in ['summary', 'plugin_version', 'system_version', 'analysis_date', 'skipped'] %}
 			<tr>
```

Comment:
we need a better solution to this problem or we will encounter it again each time a new key is added to the results dict
the real question is: how can we accomplish that without breaking the view for old analyses?

Reply:
we could save all results under the key `results` or use a prefix like `__` but this would break the view for old analyses